samples/excel/89-preview-apis/chart-trendlines.yaml

```diff
@@ -0,0 +1,200 @@
+order: 1
+id: excel-89-preview-apis-chart-trendlines-
+name: Chart trendlines
+description: 'Add, get, and format trendlines in a chart.'
+host: EXCEL
+api_set:
+    ExcelAPI: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-trendline-polynomial").click(() => tryCatch(addTrendlinePolynomial));
```

Comment:
Now that you're no longer showing a polynomial trendline:
- change name of button to:   add-trendline
- change name of function to:  addTrendline

Reply:
Good catch!   Thank you.